### PR TITLE
Reflect that Foreman 3.6 has branched

### PR DESCRIPTION
**Pull request by ekohl**
This adds links to the 3.6 branch as well as bumping the previous version number to 3.6.

<del>Currently I'm still working on the 3.6 branch, so this is still a draft</del>

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by maximiliankolb** [comment]
Change looks good to me; I cannot be sure without merging/deploying first though. If there are no other objections, feel free to merge & then we can check again.

**Comment by maximiliankolb** [comment]
> I believe satellite writers would appreciate if PRs for Foreman 2.5 - Satellite 6.10 would still be accepted as we occasionally have to edit the 6.10 docs.

Besides this PR, I assume people will review and eventually merge PRs against 2.5. But I feel like it's justified to remove it from the PR template because it requires most likely a rebase to 2.5 anyway; so no need to "advertise" it.

**Comment by ekohl** [comment]
Updated to raise it from 2.3 to 2.4 then.

**Comment by adamlazik1** [comment]
Works for me :+1: 
Changes LGTM but I am not the best person the JSON stuff.

**Comment by ekohl** [comment]
> Changes LGTM but I am not the best person the JSON stuff.

I just copied nightly and modified the version numbers.

File: web/content/release-3.6.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+=== Installation
+
+* link:/{FOREMAN_VER}/Installing_Server/index-katello.html[Installing Katello on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Installing_Proxy/index-katello.html[Installing Smart Proxy with content on RHEL/CentOS]
+
+=== Upgrading
+
+* link:/{FOREMAN_VER}/Release_Notes/index-katello.html[Release notes for Katello on RHEL/CentOS]
+* link:/{FOREMAN_VER}/Upgrading_and_Updating/index-katello.html[Upgrading and Updating Katello on RHEL/CentOS]
```
Comment (adamlazik1) [inline web/content/release-3.6.md:23]: On a second thought, this file probable should have `.adoc` suffix.

Comment (ekohl) [reply web/content/release-3.6.md:23]: Good eye! Thanks!

**Review by adamlazik1 — APPROVED** [review]

**Pull request merged**